src/pages/AdminSettings/AdminMembers/MemberList/MemberTable.tsx

```diff
@@ -143,7 +143,7 @@ function MemberTable() {
   }, [fetchNextPage, inView, hasNextPage])
 
   const tableContent = useMemo(() => {
-    if (!data?.length || !seatData) {
+    if (!data?.length || !seatData || seatData.__typename !== 'Repository') {
```

Comment:
This check should already be caught in the hook right? Or is this to prevent the "error" page from showing up

Reply:
This is just to prevent the type error with table content data. The network boundary should catch the other types.